utility/src/main/java/org/oppia/android/util/parser/html/BUILD.bazel

```diff
@@ -0,0 +1,95 @@
+"""
+General-purpose utilities for parsing & rendering HTML within the app.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "html_parser",
+    srcs = [
+        "HtmlParser.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        ":custom_bullet_span",
+        ":custom_html_content_handler",
+        ":tag_handlers",
+        "//third_party:androidx_core_core-ktx",
+        "//third_party:javax_inject_javax_inject",
+        "//utility/src/main/java/org/oppia/android/util/logging:console_logger",
+        "//utility/src/main/java/org/oppia/android/util/parser/image:url_image_parser",
+    ],
+)
+
+kt_android_library(
+    name = "tag_handlers",
+    srcs = [
+        "BulletTagHandler.kt",
+        "ConceptCardTagHandler.kt",
+        "ImageTagHandler.kt",
+        "MathTagHandler.kt",
+    ],
+    visibility = ["//:oppia_testing_visibility"],
+    deps = [
+        ":custom_html_content_handler",
+        "//utility/src/main/java/org/oppia/android/util/logging:console_logger",
+    ],
+)
+
+kt_android_library(
+    name = "custom_bullet_span",
+    srcs = [
+        "CustomBulletSpan.kt",
+    ],
+    visibility = [
+        "//:oppia_testing_visibility",
+    ],
+    deps = [
+        "//utility",
+    ],
+)
+
+kt_android_library(
+    name = "custom_html_content_handler",
+    srcs = [
+        "CustomHtmlContentHandler.kt",
+    ],
+    visibility = [
+        "//:oppia_testing_visibility",
+    ],
+    deps = [
+        "//third_party:androidx_core_core-ktx",
+    ],
+)
+
+kt_android_library(
+    name = "exploration_html_parser_entity_type",
+    srcs = [
+        "ExplorationHtmlParserEntityType.kt",
+    ],
+    visibility = [
+        "//:oppia_testing_visibility",
```

Comment:
This also seems like an API visibility library given it's used broadly in the codebase.

Reply:
when I tried to find ussages of this it shows none but when I check on the class definations they are shown.
changed it to api visibility.
![image](https://user-images.githubusercontent.com/76404817/145676444-11e0c1ea-8212-44b0-a13e-accbb629937a.png)